js/src/vector/nested.ts

```diff
@@ -40,6 +40,15 @@ export abstract class NestedView<T extends NestedType> implements View<T> {
     public toArray(): IterableArrayLike<T['TValue']> {
         return [...this];
     }
+    public find(search: T['TValue']) {
+        let index = 0;
+        for (let value of this) {
+            if (value === search) { return index; }
```

Comment:
@TheNeuralBit I saw the special-cased `every()` call in `FixedSizeListView`'s find method, so I suspect you're already aware of the question I have here: since `get(i)` on a NestedView returns a structured clone of the nested element, how do we make this work? I know in C++ they have visitors that compare vectors of the same type together, and that sounds like what we really need here.